wire-grpc-client/src/main/java/com/squareup/wire/GrpcMethod.kt

```diff
@@ -269,6 +426,16 @@ internal sealed class GrpcMethod<S : Any, R : Any>(
               && responseType == MessageSource::class.java) {
```

Comment:
you can drop all these cases right?

Reply:
right... I was thinking server uses it but GrpcMethod is for client!